ext/standard/assert.c

```diff
@@ -289,15 +283,15 @@ PHP_FUNCTION(assert_options)
 		} else {
 			RETVAL_NULL();
 		}
-		if (ac == 2) {
+		if (value) {
```

Comment:
Doesn't this prevent setting ASSERT_CALLBACK to null?

Reply:
Yes, good catch! I've changed it + added a test